src/reactTemplates.js

```diff
@@ -418,7 +421,11 @@ function convertTemplateToReact(html, options) {
             //if (options.modules === 'typescript') {
             //    defines['./' + tag.attribs.dependency] = tag.attribs.as;
             //} else {
-            defines[tag.attribs.dependency] = tag.attribs.as;
+            if ((options.mode == 'show') && (tag.attribs.mode == 'editor')) {
```

Comment:
make it a bit more generic so maybe it can be applied to the source repo as well?